### PR TITLE
Am14:  Trigger milestones and awards.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/Game.ts [hunk 1]
```diff
@@ -386,34 +386,28 @@ export class Game implements ILoadable<SerializedGame, Game> {
     }
 
     // Function to construct the board and milestones/awards list
-    public boardConstructor(boardName: BoardName, randomMA: RandomMAOptionType, hasVenus: boolean): Board {
-      const requiredQty = 5;
-      if (boardName === BoardName.ELYSIUM) {
+    public boardConstructor(boardName: BoardName, randomMA: RandomMAOptionType, hasVenus: boolean, hasAres: boolean): Board {
+      const chooseMilestonesAndAwards = function(game: Game, milestones: Array<IMilestone>, awards: Array<IAward>) {
```
Comment (bafolts) [inline src/Game.ts:390]: I would use a fat arrow function if possible. The fat arrow binds the this argument. I will enable another lint rule with PR to fix the places we aren't using it.

https://eslint.org/docs/rules/prefer-arrow-callback

**Review by bafolts — APPROVED** [review]

File: src/Game.ts (hunk 1 above)
Comment (kberg) [reply src/Game.ts:390]: Will do prior to submit - thank you!

Comment (kberg) [reply src/Game.ts:390]: Actually, I need some help here -- can you jump in and tweak this one? I'll watch and learn.

Comment (kberg) [reply src/Game.ts:390]: (Committing in the meantime.)

**Pull request merged**